theseus_gui/src/components/ui/UserSection.vue

```diff
@@ -0,0 +1,45 @@
+<script setup>
+import { Avatar } from 'omorphia'
+</script>
+
+<template>
+  <section class="user-section">
+    <Avatar size="sm" circle src="https://cdn.modrinth.com/data/AANobbMI/icon.png" />
+    <section>
+      <p id="username">OreoViking</p>
```

Comment:
this text should be pure white/black #fff or #000. there is an omorphia variable for this.